### PR TITLE
Add english prompts for GEM/wikilingua (18 languages)

### PR DESCRIPTION
**Pull request by haileyschoelkopf**
Added 5 English prompts for each of the 18 languages in GEM/wikilingua!

Will need others to write non-english prompts for this dataset if time permits.

### CODE REVIEW

**Comment by afaji** [comment]
@haileyschoelkopf I can assist the Indonesian

**Comment by haileyschoelkopf** [comment]
@afaji That would be great! Thank you

**Comment by afaji** [comment]
https://github.com/haileyschoelkopf/promptsource/pull/1/files <- Indonesian prompt for wikilingua

**Comment by stephenbach** [comment]
Looks good! Should the `article_summary_xx` prompts indicate what the language should be?

**Comment by haileyschoelkopf** [comment]
I could definitely make that change if you think it's better!

My idea was to have a prompt that didn't indicate the language (since these dataset splits all have the same source and target language) to see if it degraded performance when summary language wasn't explicitly specified, but that might not be desirable for the hackathon evals.

**Comment by stephenbach** [comment]
Yes, I think we should change it (or remove if that makes the prompt redundant with others?) It's a potentially interesting comparison to make, but balanced against the overall criteria for prompts in promptsource, I think it's better to have the task explicitly stated. CCing @awebson to see if he agrees?

**Comment by haileyschoelkopf** [comment]
Of course, makes sense! I've made the changes and explicitly stated language for the `tldr_xx` prompts as well.

File: promptsource/templates/GEM/wiki_lingua/pt/templates.yaml [hunk 1]
```diff
@@ -0,0 +1,79 @@
+dataset: GEM/wiki_lingua
+subset: pt
+templates:
+  103a066c-b175-439f-9d41-2a9563e73dad: !Template
+    answer_choices: null
+    id: 103a066c-b175-439f-9d41-2a9563e73dad
+    jinja: 'Article in Portugueses: {{source}}
```
Comment (stephenbach) [inline promptsource/templates/GEM/wiki_lingua/pt/templates.yaml:7]:
```suggestion
    jinja: 'Article in Portuguese: {{source}}
```
Fix typo.

**Review by stephenbach — APPROVED** [review]
Looks great! Thanks!

**Pull request merged**